tests/Command/AddUserCommandTest.php

```diff
@@ -84,13 +88,19 @@ public function isAdminDataProvider(): ?\Generator
      */
     private function assertUserCreated(bool $isAdmin): void
     {
+        /** @var UserRepository $repository */
+        $repository = $this->getContainer()->get(UserRepository::class);
+
         /** @var \App\Entity\User $user */
```

Comment:
this should be removed as you added the proper `@method` on the UserRepository

Reply:
Required before https://github.com/symfony/demo/commit/1c9ca0153b2a104e766929aab6fbf7d71ba8277d. Unnecessary at the moment. I’ve removed it.